mobx/lib/src/core/computed.dart

```
@@ -57,11 +61,15 @@ class ComputedValue<T> extends Atom implements Derivation {
     if (track) {
       value = _context.trackDerivation(this, _fn);
     } else {
-      try {
+      if (_context.config.disableErrorBoundaries == true) {
```

Ignore error boundary. This is part of the `ReactiveConfig`